src/test/java/io/split/android/engine/experiments/SplitParserTest.java

```diff
@@ -49,159 +42,10 @@
  */
 public class SplitParserTest {
 
-    @Test
```

Comment:
are we getting rid of these tests?

Reply:
They use StaticSegment, which does not make sense since what the sdk use are MySegment. All these tests are testing and simulating a behavior that will never happen.